src/test/java/hw5/Hooks/CucumberHooks.java

```diff
@@ -0,0 +1,17 @@
+package hw5.Hooks;

```

Comment:
Basically, this files have no relation to HW-6, so they should not be in pull request

Reply:
They are from previous pull request...(a day apart) Should I do something with this now when they are merged?

Reply:
rebased